api/docker/php/docker-entrypoint.sh

```diff
@@ -26,7 +26,6 @@ if [ "$1" = 'php-fpm' ] || [ "$1" = 'php' ] || [ "$1" = 'bin/console' ] || [ "$1
       fi
     fi
     
-    export COMPOSER_HOME="/tmp/composer"
 		composer install --prefer-dist --no-progress --no-interaction
```

Comment:
does the composer command have to  be so far to the right?

Reply:
https://github.com/ecamp/ecamp3/pull/3211/commits/419300e241fcd3a6bb9e1cb67e5afbe6311fcbe8